src/components/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './SearchForm.scss';
+
+class SearchForm extends React.Component {
+    handleChangeQuery = (e) => {
+        this.props.onQueryChange(e.target.value)
```

Comment:
You don't need to pass onQueryChange as a prop. You only need to pass initialValue to the form's input, and let SearchForm control the input state. So, you should be passing 2 props:

1. query - By the way, you can rename this to initialSearchQueryValue or something similar.
2. onSearch